tests/minimal_init.lua

```diff
@@ -134,4 +134,6 @@ require('nvim-treesitter.configs').setup({
 require('orgmode').setup({
   org_agenda_files = { base_root_path .. '/plenary/fixtures/*' },
   org_default_notes_file = base_root_path .. '/plenary/fixtures/refile.org',
+  -- Many of the tests were originally written with indent mode being enabled in mind :/
+  org_startup_indented = true,
```

Comment:
We should be able to leverage new `org_adapt_indentation` option here, right?

Reply:
Unfortunately not. `org_adapt_indentation` purely attempts to enable Virtual Indent if indent mode is on.

This enables the normal raw indents (and Virtual in some cases) that we need for many of the tests that were written with them in mind.

We actually need `org_startup_indented` set to true for *many* of the tests to work. Should I set `org_startup_indented` to `true` by default with that in mind?

Reply:
Deleted comment was not valid. Working on this now.

Reply:
We now leverage `org_adapt_indentation` by default as `org_adapt_indentation` is *enabled* by default.